Added more details staking proxy (#3882)
w3f/polkadot-wiki: docs/learn/learn-staking-advanced.md

having special permissions also called **proxy accounts**. For mode details about proxy accounts
visit the [dedicated page](./learn-proxies.md) on this wiki.

Proxy accounts are special accounts which can sign [**extrinsic calls**](#pallets-and-extrinsics)
made to specific [**pallets**](#pallets-and-extrinsics) on behalf of the proxied account. There is
thus the possibility to create staking proxy accounts that can be used to sign only extrinsic calls
to staking, session and utility pallets.

![stash-controller](../assets/stash-controller.png)

> Usually with the stash-controller setup the stash still needs to sign for bonding more funds and
> change the controller (see figure above). With a staking proxy the stash account even more isolated
> than using a controller account since one can bond more funds and change the controller using the
> staking proxy account. This does not necessarily mean that the staking proxy can sign for all
> staking-related transactions. Below we show two main configurations that affects the permissions of
> a staking proxy.

Reviewer: Grammar and punctuation fixes

```suggestion
Usually, with the stash-controller setup, the stash still needs to sign for bonding more funds and
change the controller (see figure above). With a staking proxy, the stash account is even more isolated
than a controller account since one can bond more funds and change the controller using the
staking proxy account. This does not necessarily mean that the staking proxy can sign for all
staking-related transactions. Below we show two main configurations that affect a staking proxy's permissions.
```

### Stash is not Controller

> If the stash and controller are different accounts, the staking proxy will just be used to bond more funds and change the controller. Thus the staking proxy will be used to sign for those transactions that are usually signed by the stash.

Reviewer: ```suggestion
If the stash and controller are different accounts, the staking proxy will just be used to bond more funds and change the controller. Thus the staking proxy will be used to sign for those transactions usually signed by the stash.
```

![stash-controller-stakingProxy](../assets/stash-controller-stakingProxy.png)

### Stash is also Controller

> It is not necessary to have a controller, if you have a staking proxy: the stash can also
> be the controller, and the account security will be not compromised. In this case the staking proxy will be used to sign all staking-relate transactions.

Reviewer: ```suggestion
It is not necessary to have a controller if you have a staking proxy: the stash can also
be the controller, and the account security will not be compromised. In this case, the staking proxy will be used to sign all staking-relate transactions.
```

![stash-stakingProxy](../assets/stash-stakingProxy.png)

> Note that this situation is similar to that of having the controller also as a staking proxy.

Reviewer: ```suggestion
Note that this situation is similar to having the controller as a staking proxy.
```
